src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx

```diff
@@ -246,8 +243,6 @@ const mapStateToProps = state => {
     appealsLoading: claimsV2Root.appealsLoading,
     claimsAppealsCount,
     claimsAppealsList,
-    consolidatedModal: claimsRoot.consolidatedModal,
-    show30DayNotice: claimsRoot.show30DayNotice,
     synced: claimsState.claimSync.synced,
```

Comment:
Are the 3 deleted lines above intentional and within scope?

Reply:
within scope? given that there isn't a ticket for this work, yes :-D